### PR TITLE
Reader: enable blog sticker removal from post options menu

### PR DESCRIPTION
**Pull request by bluefuton**
Following on from https://github.com/Automattic/wp-calypso/pull/15757, this PR allows Automattic users to remove blog stickers that have been set for a blog:

screen shot 2017-07-10 at 17 18 55

### To test

Pull up a site stream like:

http://calypso.localhost:3000/read/blogs/122463145

Verify that you can successfully add and remove the 'dont-recommend' and 'broken-in-reader' blog stickers.

### CODE REVIEW

File: client/blocks/reader-post-options-menu/blog-sticker-menu-item.jsx [hunk 1]
```diff
@@ -18,12 +18,9 @@ class ReaderPostOptionsMenuBlogStickerMenuItem extends React.Component {
...
 	};
 
-	addSticker = () => {
-		if ( this.props.hasSticker ) {
-			return null;
-		}
-
-		this.props.addBlogSticker( this.props.blogId, this.props.blogStickerName );
+	toggleSticker = () => {
+		const action = this.props.hasSticker ? 'removeBlogSticker' : 'addBlogSticker';
```
Comment (samouri) [inline client/blocks/reader-post-options-menu/blog-sticker-menu-item.jsx]: can we avoid the dynamic key into props?

```js
const { removeBlogSticker, addBlogSticker, blogId, blogStickerName } = this.props;

const toggle = this.props.hasSticker ?  removeBlogSticker : addBlogSticker;
toggle( blogId, blogStickerName )
```

**Comment by samouri** [comment]
Functionality checked out 👍

**Comment by jancavan** [comment]
It feels like the notification has a bit more delay when appearing/disappearing compared to when you follow/unfollow a site?

**Comment by bluefuton** [comment]
> It feels like the notification has a bit more delay when appearing/disappearing compared to when you follow/unfollow a site?

Good spot. The follow/unfollow success notice is fired as soon as you hit the button. If there's a problem, you get an additional notice informing you of the error.

For the blog sticker, we wait for the API response before displaying the notice. As it's an admin feature, I thought it was best not to say the operation is complete until it actually is (in case they navigate away from the page, for example). Thoughts @samouri?

File: client/blocks/reader-post-options-menu/blog-sticker-menu-item.jsx (hunk 1 above)
Comment (bluefuton) [inline client/blocks/reader-post-options-menu/blog-sticker-menu-item.jsx]: It's definitely neater :) There's one problem though - the linter complains about:

`'addBlogSticker' is already declared in the upper scope`

If we end up aliasing the action name to get around that, it suddenly doesn't feel as neat. What do you think?

**Comment by samouri** [comment]
@bluefuton: i actually agree with this.  we do that for follows/unfollows to _feel_ faster.  We don't care about that as much here

File: client/blocks/reader-post-options-menu/blog-sticker-menu-item.jsx (hunk 1 above)
Comment (samouri) [inline client/blocks/reader-post-options-menu/blog-sticker-menu-item.jsx]: only line that needs to change is:
```
const toggle = this.props.hasSticker ?  this.props.removeBlogSticker : this.props.addBlogSticker;
```

**Pull request merged**